docs/framework/wpf/getting-started/walkthrough-my-first-wpf-desktop-application.md

```diff
@@ -398,7 +406,7 @@ The appearance of various elements is often the same for all elements of the sam
 
     [!code-xaml[ExpenseIt#20](~/samples/snippets/csharp/VS_Snippets_Wpf/ExpenseIt/CSharp/ExpenseIt7/ExpenseReportPage.xaml#20)]
 
-    This adds styles to the <xref:System.Windows.Controls.Label> and <xref:System.Windows.Controls.Border> elements.
+    This XAML adds styles to the <xref:System.Windows.Controls.Label> and <xref:System.Windows.Controls.Border> elements.
```

Comment:
The application should be run again after this shot with a screen shot to show it's the same as the previous one.